CHANGELOG.rst

```diff
@@ -32,6 +32,8 @@ Changelog
   :attr:`~cryptography.x509.RFC822Name.value` attribute was deprecated, users
   should use :attr:`~cryptography.x509.RFC822Name.bytes_value` to access the
   raw value.
+* Added :attr:`~cryptography.x509.CertificateRevocationList.is_signature_valid`
```

Comment:
Ugh, we made it a property on CSR and a method on CRL then. Should we just change it to a property on CRL for consistency?